sqlserver/datadog_checks/sqlserver/data/conf.yaml.example

```diff
@@ -178,8 +178,8 @@ instances:
     ## @param adoprovider - string - optional - default: SQLOLEDB
     ## Choose the ADO provider.  Note that the (default) provider
     ## SQLOLEDB is being deprecated.  To use the newer MSOLEDBSQL
-    ## provider, set the adoprovider to "MSOLEDBSQL" below. You will also need
-    ## to download the new provider from
+    ## provider, set the adoprovider to "MSOLEDBSQL" below or "MSOLEDBSQL19" for version 19 of the driver.
+    ## You will also need to download the new provider from
```

Comment:
You will need to update this file as well https://github.com/DataDog/integrations-core/blob/master/sqlserver/assets/configuration/spec.yaml

The `[conf.yaml.example](https://github.com/DataDog/integrations-core/pull/12916/files#diff-41741d934cef4d21a79c745bc5e6ce308094c482c629363a31fcac614c0e2e3d)` is generated from the `spec.yaml` file using the `ddev validate config sqlserver -s` command

Reply:
Raised a PR for the other file as well:
https://github.com/DataDog/integrations-core/pull/12917

Assuming I can close this PR since its generated from the spec.yaml file?

Reply:
You need to modify both files in the same PR :)

Reply:
Done!

Reply:
Thank you!